lib/tasks/mastodon.rake

```diff
@@ -19,6 +19,39 @@ namespace :mastodon do
     puts "User #{email} confirmed."
   end
 
+  desc 'Add a user by providing thier email, username and initial password. ' \
+       'The user would recieve a confirmation mail and then would be able to reset his password for logging in.'
```

Comment:
Grammar nits on this, but instead:

If you're expecting them to reset their password, use `SecureRandom.hex` for the password instead of prompting the admin.
Then replace the Enter Password loop with a Confirm Y/N.